source/includes/example-connect-onprem-to-atlas.rst

```diff
@@ -0,0 +1,79 @@
+Gather Connection Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The source cluster, ``cluster0``, is hosted on the following servers
+and ports:
+
+- clusterOne01.fancyCorp.com:20020
+- clusterOne02.fancyCorp.com:20020
+- clusterOne03.fancyCorp.com:20020
+
+The destination Atlas cluster, ``cluster1``, is hosted on the following
+servers and ports:
+
+- cluster2Name-01.abc123.com:27017
+- cluster2Name-02.abc123.com:27017
+- cluster2Name-03.abc123.com:27017
+
+There is an administrative user, ``clusterAdmin`` configured on each
+cluster with password, ``superSecret``.
+
+Connect the Source and Destination Clusters with ``mongosync``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The generic connection string format for the self-managed cluster is: 
+
+.. code-block:: shell
+
+   mongodb://<user>:<password>@<ip-address>:<port>,<ip-address>:<port>,<ip-address>:<port>
+
+The generic connection string format for the Atlas cluster is: 
+
+.. code-block:: shell
```

Comment:
For this code block, set the format to text since it's a raw example and doesn't need syntax highlighting.

Reply:
I'm still seeing these set to shell.  They should be `text` or `none`.